bin/runtests

```diff
@@ -191,7 +191,7 @@ function get_bundles($target = null)
 
 function get_test_console_path()
 {
-    $consolePath = __DIR__ . '/../src/Sulu/Bundle/TestBundle/Resources/app/console';
+    $consolePath = __DIR__ . '/../packages/article/Tests/Application/bin/adminconsole'; // to init the database with complete schema was using currently article bundle
```

Comment:
this may should be changed in future init the database correctly per bundle instead only once but currently go with this solution